openmc/weight_windows.py

```diff
@@ -443,7 +449,7 @@ def from_hdf5(cls, group, meshes) -> WeightWindows:
         )
 
 
-def wwinp_to_wws(path) -> List[WeightWindows]:
+def wwinp_to_wws(path: Union[str, pathlib.Path]) -> List[WeightWindows]:
```

Comment:
In the checkvalue.py module, we have an alias for this called `PathLike` that is used elsewhere